webui/react/src/services/apiConfig.ts

```diff
@@ -873,7 +873,7 @@ export const getExpTrials: DetApi<
   postProcess: (response) => {
     return {
       pagination: response.pagination,
-      trials: response.trials.map(decoder.decodeV1TrialToTrialItem),
+      trials: response.trials.map((t) => decoder.decodeV1TrialToTrialItem(t)),
```

Comment:
**nit:** aren't these two lines the same?

Reply:
Not exactly, the `decodeV1TrialToTrialItem` now takes 2 parameters so previous way of calling it throws an error